__tests__/app.test.js

```diff
@@ -216,3 +216,74 @@ describe('POST /api/articles/:article_id/comments', () => {
     })
  })
 })
+
+describe('PATCH /api/articles/:article_id', () => {
+  test('200: Allows a user to update votes on an article POSITIVE NUMBER', () => {
+      const newVote = {inc_votes: 100};
+      return request(app)
+      .patch('/api/articles/1')
+      .send(newVote)
+      .expect(200)
+      .then(({body}) => {
+        expect(body.votes).toEqual(
+          expect.objectContaining(
+            {
+              title: "Living in the shadow of a great man",
+              topic: "mitch",
+              author: "butter_bridge",
+              body: "I find this existence challenging",
+              created_at: expect.any(String),
+              votes: 200,
+              article_img_url:
+                "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            },
+          )
+        )
+      })
+   })
+   test('200: Allows a user to update votes on an article NEGATIVE NUMBER', () => {
+    const newVote = {inc_votes: -100};
+    return request(app)
+    .patch('/api/articles/1')
+    .send(newVote)
+    .expect(200)
+    .then(({body}) => {
+      expect(body.votes).toEqual(
+        expect.objectContaining(
+          {
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: expect.any(String),
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+        )
+      )
+    })
+ })
+   test('404: throws an error when posting to an article that doesnt exist', () => {
+    const newVote = {inc_votes: 100};
+    return request(app)
+    .patch('/api/articles/5000')
+    .send(newVote)
+    .expect(404)
+    .then(({body}) => {
+      const {msg} = body;
+      expect(msg).toBe('not found')
+    })
+ })
+ test('400: throws an error when passing in an invalid vote count', () => {
+  const newVote = {inc_votes: 'HELLO'};
+  return request(app)
+  .patch('/api/articles/5000')
+  .send(newVote)
+  .expect(400)
+  .then(({body}) => {
+    const {msg} = body;
+    expect(msg).toBe('bad request')
+  })
+})
```

Comment:
This is optional but one worth considering for a stretch goal. 
This patch _could_ be used to change any key on the article. What if there was a spelling mistake in a title or a correction needed on the body?
How would you handle patch requests with more than one key, would you want it to error or ignore it?